Makefile

```diff
@@ -30,7 +31,7 @@ release: release-darwin-aarch64 release-darwin-x86_64 release-linux-arm64 releas
 	cp artifacts/all/libtokenizers.linux-x86_64.tar.gz artifacts/all/libtokenizers.linux-amd64.tar.gz
 
 test: build
-	@go test -v ./... -count=1
+	@go test -ldflags="-extldflags '-L./'" -v ./... -count=1
```

Comment:
The build step copies the architecture built library to the `PWD` so we just tell the linker to search the `PWD` for the library to link in tests.